test/testvm.py

```diff
@@ -971,15 +976,34 @@ def wait_reboot(self):
             raise Failure("system didn't reboot properly")
         self.wait_user_login()
 
-    def wait_boot(self, wait_for_running_timeout = 120):
+    def wait_boot(self, wait_for_running_timeout = 120, allow_one_reboot=False):
```

Comment:
The flag should be called expect_one_reboot, because the code below expects it, doesn't just allow it to happen.